proto/raft_cmdpb.proto

```diff
@@ -99,6 +108,7 @@ message Request {
     DeleteRangeRequest delete_range = 8;
     IngestSSTRequest ingest_sst = 9;
     ReadIndexRequest read_index = 10;
+    FlashbackToVersionRequest flashback_to_version = 11;
```

Comment:
why add to raft cmd, it should add to tikv server API. And all the changes are proposed as delete in raft cmd.